docs/api/supervisor/endpoints.md

```
@@ -1519,6 +1519,7 @@ Returns information about the docker instance.
 key | type | description
 -- | -- | --
 version | string | The version of the docker engine
+enable_ipv6 | bool | Enabled/Disabled IPv6 for containers
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)